.murdock

```diff
@@ -1,12 +1,23 @@
 #!/bin/sh
 
 # uncomment and change this to limit builds, e.g.,
-#export BOARDS="samr21-xpro native"
+export BOARDS="samr21-xpro native"
 # and / or
-#export APPS="examples/hello-world tests/unittests"
+export APPS="examples/hello-world tests/unittests"
+
+# this configures boards that are available via pifleet
+case "${CI_MURDOCK_PROJECT}" in
+    riot)
+        : ${TEST_BOARDS_AVAILABLE:="esp32-wroom-32 samr21-xpro"}
+        ;;
+    riot-staging)
+        : ${TEST_BOARDS_AVAILABLE:=""}
+        ;;
```

Comment:
Is there a plan to add boards to that instance in the future or why is there a separate branch for `riot-staging`?

Reply:
(it is identical to the default branch, that was why I was asking)

Reply:
yes... we might want to test things with boards there I guess.

That makes me thinking if this should be configured here at all.

Reply:
ok, so to pass that info from the new frontend, that would need a restart, stopping current builds and clearing the queue.

I'd like to keep the riot-staging even if it is identical, so we see right away where and now to add board(s) there.